lib/core/src/lib.rs

```diff
@@ -131,21 +128,23 @@ mod example_03 {
             let fonts_iter    = FONT_NAMES.iter().map(font_creator);
             let mut fonts     = fonts_iter.collect::<Box<[FontRenderInfo]>>();
 
-            let all_cases     = iproduct!(0..fonts.len(), SIZES.iter());
+            let all_cases     = iproduct!(0..fonts.len(), 0..SIZES.len());
 
-            for (i, (font, size)) in all_cases.enumerate() {
+            for (font, size) in all_cases {
 
-                let line_position = nalgebra::Vector2::new(-0.95, 0.9 - 0.064*(i as f64));
+                let x = -0.95 + 0.6 * (size as f64);
+                let y = 0.90 - 0.45 * (font as f64);
```

Comment:
I did not notice it earlier - why do we use such hardcoded literals here? If we have to, please refactor them to vars.

Reply:
Well, this is code for an example, and it is well-written anyway. Soon I think I remove this example and just create a test with html like Danilo did